jobs/rocoto/gempak.sh

```diff
@@ -12,14 +12,20 @@ IFS=', ' read -r -a fhr_list <<< "${FHR_LIST}"
 
 export FHR3 jobid
 for fhr in "${fhr_list[@]}"; do
-	FHR3=$(printf '%03d' "${fhr}")
-	jobid="${job}_f${FHR3}.$$"
-	###############################################################
-	# Execute the JJOB
-	###############################################################
-	"${HOMEgfs}/jobs/J${RUN^^}_ATMOS_GEMPAK"
-	status=$?
-	[[ ${status} -ne 0 ]] && exit "${status}"
+    FHR3=$(printf '%03d' "${fhr}")
+    jobid="${job}_f${FHR3}.$$"
+    ###############################################################
+    # Execute the JJOB
+    ###############################################################
+    if [[ "${RUN}" == "gefs" ]]; then
+        echo "add J-script here for GEFS."
+    else
+        "${HOMEgfs}/jobs/JGFS_ATMOS_GEMPAK"
+        err=$?
+        if [[ ${err} -ne 0 ]]; then
+          exit "${err}"
+        fi
+    fi
```

Comment:
```suggestion
    "${HOMEgfs}/jobs/J${RUN^^}_ATMOS_GEMPAK"
    err=$?
    if [[ ${err} -ne 0 ]]; then
      exit "${err}"
    fi
```
Please revert the indent changes in this script.

Reply:
I am not sure if `JGEFS_ATMOS_GEMPAK` will be needed or not.  It is possible that `JGFS_ATMOS_GEMPAK` can be renamed to `JGLOBAL_ATMOS_GEMPAK`
We can leave that for the next person who enables that job